docs/operate/customize/dois.md

```diff
@@ -130,24 +136,20 @@ INVENIO_RDM_PERSISTENT_IDENTIFIERS = {
     },
 }
 ```
-You [can view the default configuration in invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/e64dd0b81757a391584e63d162d5e6caf6780637/invenio_rdm_records/config.py#L322)
+You [can view the default configuration in invenio-rdm-records](https://github.com/inveniosoftware/invenio-rdm-records/blob/7869cb3242a7feb311f97b633af095edeff4de76/invenio_rdm_records/config.py#L393C1-L410C2)
 
-### Optional DOIs
+
+### Optional DOI User Interface and Advanced Configuration
 
 _Introduced in InvenioRDM v13_
 
-You can configure InvenioRDM to allow users to choose whether or not to register a DOI when uploading a record.
+InvenioRDM now includes a user interface to allow users to choose whether or not to register a DOI when uploading a record.
 
 ![Optional DOIs](imgs/optional-dois.jpg)
 
-To enable this feature, configure the following in your `invenio.cfg`:
-
-```python
-### Do not require DOIs for record and parent
-RDM_PERSISTENT_IDENTIFIERS["doi"]["required"] = False
-RDM_PARENT_PERSISTENT_IDENTIFIERS["doi"]["required"] = False
-RDM_PERSISTENT_IDENTIFIERS["doi"]["ui"]["default_selected"] = "not_needed"  # "yes", "no" or "not_needed"
-```
+This will display when both main and parent DOIs are configured as
+optional. You can also set the default button selection with the
+"doi" "ui" "default_selected" option, which can be "yes", "no" or "not_needed"
```

Comment:
The code snippet was a good summary of the combination of settings needed. Is it not accurate anymore?

Reply:
It doesn't work if you don't have the variables defined elsewhere in your configuration file. I like the code snippets to work if you copy and paste them. Since the optional DOI config was already done in the earlier section, this one isn't really needed.